priv/modules/greeter.xq

```diff
@@ -0,0 +1,23 @@
+ element html {
+    element head {
+    element title { 'Now Flying Xqerl'},
+    element link { 
+      attribute rel {"stylesheet"},
+      attribute href { "/assets/styles/sakura.css"},
+      attribute type {"text/css"}
+    },    
+    element link { 
+      attribute rel {"icon"},
+      attribute href { "/assets/images/favicon.ico"},
+      attribute sizes {"any"}
+    }
+  },
+  element body{ 
+    element img { 
+     attribute src { "/assets/images/logo_m.png" },
+     attribute alt { "xqerl logo" }
+     },
+    element h1 { 'You Are Now Flying Xqerl'  },
```

Comment:
❤️

Reply:
![NowFlyingXqerl](https://user-images.githubusercontent.com/1757792/144516472-6e42d8a6-0a39-4d19-b370-8d25f01961c4.png)

Reply:
Most probably the png image background color needs to be transparent